pom.xml

```diff
@@ -378,36 +377,16 @@
             <artifactId>brapi-java-client</artifactId>
             <version>${brapi-java-client.version}</version>
         </dependency>
-        <dependency>
```

Comment:
I assume tika is pulling all these in?

Reply:
Yes, maven pulls those in to build tika as they are dependencies of tika. I don't know how common or advisable it is to hard-code transitive dependencies with maven projects, I used trial and error to determine that maven did the right thing for all the transitive dependencies besides apache commons compress, which I hard-coded. I think the issue might have been that two primary dependencies of bi-api both depend on apache commons compress, and maven was defaulting to using a version that didn't work with tika.